README.md

```diff
@@ -1046,8 +1047,6 @@ A collection of awesome things regarding React ecosystem.
 * [Isomorphic Development with JavaScript](https://www.manning.com/books/isomorphic-development-with-javascript)
 * [React in Action](https://www.manning.com/books/react-in-action)
 * [React Quickly](https://www.manning.com/books/react-quickly)
-* [The Road to learn React](https://roadtoreact.com/course-details?courseId=THE_ROAD_TO_LEARN_REACT)
```

Comment:
where are they duplicated?

Reply:
Besides these two, there is a different link to "The Road to Learn React" under books. That seems sufficient to me.